src/main/java/frc/robot/RobotContainer.java

```diff
@@ -266,6 +266,7 @@ private void setupAltOperatorController() {
     }
 
     private void setupPitController() {
+        pitController.a().whileTrue(CommandFactory.scoreInBarge(swerve, elevator, algae));
```

Comment:
This needs to be replaced with a web controller integration. So it either needs to be part of this PR or a new issue created to follow up.